x/wasm/types/params_test.go

```diff
@@ -89,6 +110,27 @@ func TestValidateParams(t *testing.T) {
 			},
 			expErr: true,
 		},
+		"reject invalid address in any of addresses": {
```

Comment:
Can you add test case for AnyOf with empty list? That should also fail (it makes no sense, likely typo)